srcpkgs/rsvd-user/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'rsvd-user'
+pkgname=rsvd-user
+version=1.0
+revision=1
+build_style="meta"
+make_dirs="/etc/rsvd-user 755 root root"
+depends="runit"
+short_desc="User runsvdir daemons framework"
+maintainer="danoloan10 <danoloan10@tutanota.com>"
+license="Public Domain"
+homepage="danoloan.es"
+distfiles="https://github.com/danoloan10/rsvd-user/archive/v${version}.tar.gz"
+checksum=fc17a479fba95612c8bf3722ebe5b57a15430009694ca1bd7c3b7e0eb8dcc397
+
+do_install() {
+	cp ${wrksrc}/etc/* /etc/ -r
```

Comment:
Have you built this package? `do_install` is run from inside the wrksrc dir, so it shouldn't need `${wrksrc}`. That command is only copying stuff from your builddir into the chroot's `/etc` (so it isn't getting packaged). You might be looking for the `vcopy` or `vinstall` commands instead (they are explained in the Manual).